app/assets/stylesheets/base.css.scss

```diff
@@ -15,12 +15,12 @@ body {
   h1 {
     font-family: $headers;
     font-weight: 700;
-    font-size: 3vw;
+    //font-size: 3vw;
```

Comment:
For all these commented-out elements, let's just remove them, unless you think there is value in retaining them; if so, perhaps some descriptive comments would help.